portal/src/styles/variables.scss

```diff
@@ -0,0 +1 @@
+$body-font-family: 'Source Sans Pro';
```

Comment:
is this used anywhere?

Reply:
Yes, it's the way that Vuetify gets an external font and it will be possible to override other SASS variables. Otherwise the font would be Roboto.
This solution was provided by: https://stackoverflow.com/questions/45598884/change-default-font-in-vuetify#answer-61969663, which is not the accepted answer, but it seemed to be the simplest and most direct solution.

Reply:
indeed, nice solution